lib/hydra/works/models/concerns/generic_work_behavior.rb

```diff
@@ -0,0 +1,61 @@
+module Hydra::Works
+  module GenericWorkBehavior
```

Comment:
This should be Hydra::Works::WorkBehavior, not Hydra::Works::GenericWorkBehavior. Hydra::Works::GenericWork is a Generic model which uses the Hydra::Works::WorkBehaviors.